_posts/2023-04-05-ml-deployment-status.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title:  "You can now see ML model status in OpenSearch Dashboards"
+authors:
+- wonglam
+- yych
+- naarcha
+date: 2023-04-05
```

Comment:
Publishing date will need to be updated. Please let me know the new scheduled date.

Reply:
Publishing date is at the discretion of Marketing. Please update as appropriate.